action/receipt.go

```diff
@@ -163,6 +166,20 @@ func (receipt *Receipt) SetExecutionRevertMsg(revertReason string) *Receipt {
 	return receipt
 }
 
+// UpdateIndex updates the index of receipt and logs
+func (receipt *Receipt) UpdateIndex(txIndex, logIndex uint32) (uint32, uint32) {
```

Comment:
don't pass and return same var. This can be avoided by using `*uint32`

Reply:
after discussing, should not return the new index